calibration/CTDMOG/CGINS-CTDMOG-13594__20170406.csv

```diff
@@ -22,4 +22,4 @@ serial,name,value,notes
 37-13594,CC_ptempa0,1.539212e+002,
 37-13594,CC_ptempa1,-6.678808e-002,
 37-13594,CC_ptempa2,3.192585e-007,
-37-13594,CC_wbotc,1.5486e-007,
+37-13594,CC_wbotc,1.548578e-07,
```

Comment:
Looks good.

Reply:
Looks good (CD and JL) verified with QCT